Tasks/NuGetCommandV2/nugetpublisher.ts

```diff
@@ -466,4 +483,37 @@ function getAccessToken(isInternalFeed: boolean): string{
     }
 
     return accessToken;
-}
\ No newline at end of file
+}
+
+async function tryServiceConnection(endpoint: EndpointCredentials, feed: any) : Promise<boolean>
+{
+    // Create request
+    const request = new WebRequest();
+    const token64 = Buffer.from(`${endpoint.username}:${endpoint.password}`).toString('base64');
+    request.uri = endpoint.endpoint;
+    request.method = 'GET';
+    request.headers = {
+        "Content-Type": "application/json",
+        "Authorization": "Basic " + token64
+    };
+
+    return await sendRequest(request).then((response: WebResponse) => {
+        if(response.statusCode == 200) { // Not sure if we actually need this null check here..
```

Comment:
nit: contrary to the comment, there's no null check here...

Reply:
It was in reference to the following if(response.body) line, which is kind of a null check (?) Should a better null check be added here? Do I even need `if (response.body)` or is it a safe assumption that a response will include a body

Reply:
Ah, at least move the comment to the next line. End-of-line comments generally refer to what's to their left. The check itself is fine.